Scripts/Knight.gd

```diff
@@ -68,14 +68,23 @@ func finish_rolling() -> void:
 
 func finish_attacking() -> void:
 	is_attacking = false
+	can_damage = false
+	attacks_left = 0
 	
+func attack_active():
```

Comment:
Method name is ambigious. Perhaps activate_attack(). Since attack_active() can be read as a question